src/layout/type.ts

```diff
@@ -0,0 +1,7 @@
+import { COMMON_ROUTES_NAMES, GARSON_ROUTES_NAMES, ADMIN_ROUTES_NAMES } from '@/routing'
+
+export interface NavMenu {
```

Comment:
not necessary to change

it is not necessary to create this in a separate file if it's not too big and you only use it once. 
it i good you did it, but I wonder what's the possible other content of this file could be in the future